packages/admin-ui/ui/src/components/molecules/language-menu/index.tsx

```diff
@@ -0,0 +1,47 @@
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu"
```

Comment:
todo: Design will be finalized today, so let's make sure to include this before we merge

Reply:
Maybe already include a "help us translate" ?